### PR TITLE
Update Room.java

### PR DESCRIPTION
**Pull request by mst3k4ever**
Comment edits.

### CODE REVIEW

File: Android--EN/Room.java [hunk 1]
```diff
@@ -1149,13 +1149,13 @@ public void disableDeviceInputs(final boolean disableOperations) {
...
      *
-     * After calling this method, the SDK delays sending the whiteboard contents of the local user with the remote users per the set time.
+     * After calling this method, the SDK delays sending the whiteboard contents of the local user to the remote users per the set time.
      *
      * This method helps synchronize the whiteboard contents with audio and video in CND live streaming.
      *
      * @note This method does not delays the content the local users see, that is, when the local user writes or draws on the whiteboard, they see the content on their own whiteboard immediately.
      *
-     * @param delaySec The delay time in seconds.
+     * @param delaySec The delay time in seconds. // TODO William Not milliseconds? It looks like it's being multiplied by 1000? Please doublecheck.
```
Comment (TingChen2020) [inline Android--EN/Room.java]: The delay time that you pass in is the time in seconds. The SDK will multiply the set value by 1000 to turn it to time in milliseconds.

File: Android--EN/Room.java [hunk 2]
```diff
@@ -912,14 +912,14 @@ public void onValue(String result) {
...
-     * Insert multiples scenes under the specified scene directory.
+     * Inserts multiples scenes under the specified scene directory.
      *
      * @note
      * This method does not switch the whiteboard scene to any of the newly inserted scenes.
      * You need to call {@link #setScenePath(String) setScenePath(} to switch to one of the newly inserted scenes.
      *
      *
-     * @param dir    The path of the scene directory, which must starts with `/` and cannot be the path of a scene. For example, `"/math"`.
+     * @param dir    The path of the scene directory, which must starts with `/` and cannot be the path of a scene. For example, `"/math"`. // TODO Willima I am unsure if these parameter lists are supposed to be aligned or not; sometimes they are, sometimes they aren't. I'm leaving them "as is" for now.
```
Comment (TingChen2020) [inline Android--EN/Room.java]: The param lists are aligned after automatic formatting by an IDE (the developers set their IDE to do so). However, that is not necessary when we use Doxygen to generate the documentation. We just need to ensure that there is a space between the param name and the description.

File: Android--EN/Room.java (hunk 1 above)
Comment (mst3k4ever) [reply Android--EN/Room.java]: Okay!

File: Android--EN/Room.java (hunk 2 above)
Comment (mst3k4ever) [reply Android--EN/Room.java]: Got it. Thanks!

**Pull request merged**